src/context/monika-flags.ts

```diff
@@ -52,7 +52,7 @@ export const monikaFlagsDefaultValue: Pick<
   config: getDefaultConfig(),
   'config-filename': 'monika.yml',
   'config-interval': 900,
-  'follow-redirects': 1,
+  'follow-redirects': 3,
```

Comment:
IMO it is still too shallow. For comparison, curl default is 50. And Axios default is 5. Need to wait others opinion on this.
More info: https://curl.se/docs/manpage.html#--max-redirs and https://axios-http.com/docs/req_config

Reply:
Sure, what should . Also interesting, axios README.md says their default is 21 [here](https://github.com/axios/axios/blob/21a5ad34c4a5956d81d338059ac0dd34a19ed094/README.md?plain=1#L475)

There may have been attempts to fix timeouts in neosense by limitting to maxRedirect to 1.

Reply:
setting it to 21 as per axios github docs